api.go

```diff
@@ -135,6 +135,16 @@ func (api *API) GetCode(account AccountName) (out *GetCodeResp, err error) {
 	return
 }
 
+func (api *API) GetCodeHash(account AccountName) (out SHA256Bytes, err error) {
+	resp := GetCodeHashResp{}
+	if err = api.call("chain", "get_code_hash", M{"account_name": account}, &resp); err != nil {
+		return
+	}
+
+	buffer, err := hex.DecodeString(resp.CodeHash)
+	return SHA256Bytes(buffer), err
```

Comment:
maybe we should rename that.. SHA256Bytes doesn't really make sense.. SHA256Hash would be better, or Checksum256 like the C++ code.

Reply:
/non-related nor blocking

Reply:
That will be a heavy refactor for all users of the library I think. I will add a second issue for this.